openedx/adg/lms/webinars/admin.py

```diff
@@ -71,52 +75,64 @@ def get_fields(self, request, obj=None):
         """
         fields = super().get_fields(request, obj)
         if not obj:
-            fields.remove('send_update_emails_to_registrants')
+            fields.remove(SEND_UPDATE_EMAILS_FIELD)
         return fields
 
     def save_related(self, request, form, formsets, change):
```

Comment:
This will also allow us to document different sections of the code via the docstring of their helpers.

Reply:
There is only one line of code corresponding to `removed_members` that lies at line 97 that can be moved to a new function, hence it would give us no benefit.

I thought of moving this code related to `new_members` to a new file:

```
            if new_members:
                WebinarRegistration.create_team_registrations(new_members, webinar)

                new_member_emails = [user.email for user in new_members]
                schedule_webinar_reminders(new_member_emails, webinar.to_dict())

                webinar_invitees_emails += new_member_emails
```

But in that case too, only two lines of code, i.e. 

```
WebinarRegistration.create_team_registrations(new_members, webinar)
schedule_webinar_reminders(new_member_emails, webinar.to_dict())
```

will be passed to a new function. I can do that if you want me to.

Reply:
That was just an example. I only meant we can add different sections of the code in different functions.
However, if you believe it doesn't really help us in terms of readability, then we can skip it.

Reply:
It contains a lot of `if`s and the control flow changes a lot. That's where I was coming from.

Reply:
I brainstormed a lot of different ways because I felt the same, but there are things needed at the start, middle, and end of the function and it is not as such that we can move certain independent sections. There are no independent sections, all are required in save_related. By the way, doing that may require a complete revamp of the code, while this ticket was only meant to be a bug fixing ticket. In any case, if you can identify an independent section that can be moved to a sub-function without breaking the logic, I'll move that. I did create 4 new helper methods, but it could not be further broken down.

Reply:
That makes sense!
Thank you for the explanation.